server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -103,10 +110,16 @@ export function create(
      * Creates a new tenant
      */
     router.post("/tenants/:id?", (request, response) => {
-        const tenantId = getParam(request.params, "id");
+        const tenantId = getParam(request.params, "id") || getRandomName("-");
+        const tenantStorage: ITenantStorage = request.body.storage || null;
+        const tenantOrderer: ITenantOrderer = request.body.orderer || null;
+        const tenantCustomData: ITenantCustomData = request.body.customData || null;
```

Comment:
For these three lines, I would use ternary operator. For example:

const tenantStorage: ITenantStorage = request.body.storage ? request.body.storage || null;

Reason is, null itself is falsy. So '||' seems incorrect since the fallback is also false

Reply:
Also you probably want {} for customData instead of null.